priv/repo/migrations/20161209192504_create_task_list.exs

```diff
@@ -0,0 +1,66 @@
+defmodule CodeCorps.Repo.Migrations.CreateTaskList do
+  use Ecto.Migration
+  import Ecto.Changeset
+  import Ecto.Query
+
+  alias CodeCorps.Project
+  alias CodeCorps.Repo
+  alias CodeCorps.Task
+  alias CodeCorps.TaskList
+
+  def change do
+    create table(:task_lists) do
+      add :name, :string
+      add :rank, :integer
```

Comment:
We could certainly change the name, but I think that `rank` is a common term when referring to sorting and sorting algorithms.